en/setup/dev_env_linux_centos.md

```diff
@@ -37,7 +37,7 @@ arm-none-eabi-gcc --version
 If you receive the following message
 
 ```sh
-bash: gcc-arm-none-eabi-4_7-2014q2/bin/arm-none-eabi-gcc: /lib/ld-linux.so.2: bad ELF interpreter: No such file or directory
+bash: gcc-arm-none-eabi-4_7-2014q2/bin/arm-none-eabi-gcc: /lib/ld-linux.so.2: bad ELF interpreter: No such file or directory 
```

Comment:
@dagar I can imagine that I **could** replace the source path to the current gcce - e.g. with something like:
``` 
gcc-arm-none-eabi-7-2017-q4-major/bin/arm-none-eabi-gcc: /lib/ld-linux.so.2: bad ELF interpreter: No such file or directory 
````
But how can I tell if this is still valid?

Reply:
If I delete the support libraries above then this won't be needed because it is only relevant to them.